packages/math-input/package.json

```diff
@@ -42,8 +42,7 @@
     "dependencies": {
         "@khanacademy/keypad-context": "workspace:*",
         "@khanacademy/perseus-core": "workspace:*",
-        "@khanacademy/perseus-utils": "workspace:*",
-        "mathquill": "file:../../vendor/mathquill-v1.0.0.tgz"
```

Comment:
We move this to be in `devDependencies` because we now bundle everything from mathquill _into_ `@khanacademy/math-input`. This relieves client apps from needing to depend on `mathquill` directly (most client apps don't use `mathquill` directly if they're using `@khanacademy/math-input`)